package.json

```diff
@@ -143,6 +143,7 @@
     "concurrently": "^8.2.0",
     "cookie-session": "^2.0.0",
     "cypress": "^12.14.0",
+    "cypress-axe": "^1.4.0",
```

Comment:
Add the dependency to `package.json`
(the massive change to`package-lock.json` above is normal - this always happens when a change is made to the dependencies - it basically re-writes the whole file)